packages/autorest.python/autorest/codegen/models/parameter.py

```diff
@@ -97,6 +97,15 @@ def __init__(
             self.location == ParameterLocation.HEADER
             and self.wire_name.lower() in SPECIAL_HANDLE_HEADERS
         )
+        self.is_special_skip_header: bool = (
+            self.location == ParameterLocation.HEADER
+            and self.wire_name.lower()
+            in self.code_model.options["skip_special_headers"]
+        )
+
+    @property
+    def is_special_header(self) -> bool:
```

Comment:
I prefer not to add this attribute to combine two different header logic.

Reply:
updated.